README.md

```diff
@@ -49,9 +61,20 @@ Please read [CONTRIBUTING.md](https://github.com/PointCloudLibrary/pcl/blob/mast
 
 Issues
 ------
-For general questions on how to use the PCL, please use the [pcl-users](http://www.pcl-users.org/) mailing list (do not forget to subscribe before posting).
+For general questions on how to use the PCL, please choose one of the following alternatives instead:
```

Comment:
I would switch the order. Start with "To report issues...", then "For general questions...".

Plus the same comment regarding link and spelling.

Reply:
PTAL